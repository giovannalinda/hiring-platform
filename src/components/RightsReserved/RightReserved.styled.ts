import styled from 'styled-components'
import { theme } from '~/styles'

export const Container = styled.div`
  border-top: 1px solid ${theme.colors.purple50};
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  border-bottom: 1px solid ${theme.colors.purple50};
  color: ${theme.colors.gray500};
  padding: ${theme.space[6]};
`
