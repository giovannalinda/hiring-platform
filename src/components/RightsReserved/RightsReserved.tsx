import * as S from './RightReserved.styled'

export function RightsReserved() {
  return (
    <S.Container>
      <span> Â© 2022 Hiring Platform. All rights reserved</span>
    </S.Container>
  )
}
