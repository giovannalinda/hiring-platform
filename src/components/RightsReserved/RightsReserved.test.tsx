import { render } from '~/__helpers__/app-tests'

import { RightsReserved } from './RightsReserved'

describe('RightsReserved', () => {
  it('should match snapshot', () => {
    const { container } = render(<RightsReserved />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
