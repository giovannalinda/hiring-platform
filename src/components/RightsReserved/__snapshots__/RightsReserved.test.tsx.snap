// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RightsReserved should match snapshot 1`] = `
<div
  class="sc-bczRLJ ifbMZa"
>
  <span>
     Â© 2022 Hiring Platform. All rights reserved
  </span>
</div>
`;
